### PR TITLE
Added some more comments to volume.py in maths folder

### PR DESCRIPTION
**Pull request by advik-student-dev**
added some more comments (to formulas which need it) which make the code more readable and understandable. might make a list of all the formulas on the top, later

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/volume.py [hunk 1]
```diff
@@ -2,11 +2,14 @@
 Find Volumes of Various Shapes.
 Wikipedia reference: https://en.wikipedia.org/wiki/Volume
```
Comment (cclauss) [inline maths/volume.py:3]:
```suggestion
Find the volume of various shapes.
* https://en.wikipedia.org/wiki/Volume
* https://en.wikipedia.org/wiki/Spherical_cap
```

File: maths/volume.py [hunk 2]
```diff
@@ -2,11 +2,14 @@
...
 """
+
+# Imports
```
Comment (cclauss) [inline maths/volume.py:6]:
```suggestion
```

File: maths/volume.py [hunk 3]
```diff
@@ -2,11 +2,14 @@
...
 from __future__ import annotations
 
 from math import pi, pow
 
 
+# Functions for calculating volumes of shapes
```
Comment (cclauss) [inline maths/volume.py:12]:
```suggestion
```

File: maths/volume.py [hunk 4]
```diff
@@ -30,8 +33,7 @@ def vol_cube(side_length: int | float) -> float:
 
 def vol_spherical_cap(height: float, radius: float) -> float:
     """
-    Calculate the Volume of the spherical cap.
-    :return 1/3 pi * height ^ 2 * (3 * radius - height)
+    Calculate the Volume of the spherical cap
```
Comment (cclauss) [inline maths/volume.py:36]:
```suggestion
    Calculate the volume of the spherical cap.
```

File: maths/volume.py [hunk 5]
```diff
@@ -47,6 +49,7 @@ def vol_spherical_cap(height: float, radius: float) -> float:
         ...
     ValueError: vol_spherical_cap() only accepts non-negative values
     """
+    # Volume - 1/3 pi * height squared * (3 * radius - height)
     if height < 0 or radius < 0:
         raise ValueError("vol_spherical_cap() only accepts non-negative values")
     return 1 / 3 * pi * pow(height, 2) * (3 * radius - height)
```
Comment (cclauss) [inline maths/volume.py:55]:
```suggestion
    if height < 0 or radius < 0:
        raise ValueError("vol_spherical_cap() only accepts non-negative values")
    # Volume is 1/3 pi * height squared * (3 * radius - height)
    return 1 / 3 * pi * pow(height, 2) * (3 * radius - height)
```

File: maths/volume.py [hunk 6]
```diff
@@ -261,6 +264,7 @@ def vol_sphere(radius: float) -> float:
         ...
     ValueError: vol_sphere() only accepts non-negative values
     """
+    # Volume - radius cubed * pi * 4/3
     if radius < 0:
         raise ValueError("vol_sphere() only accepts non-negative values")
     return 4 / 3 * pi * pow(radius, 3)
```
Comment (cclauss) [inline maths/volume.py:270]:
```suggestion
    if radius < 0:
        raise ValueError("vol_sphere() only accepts non-negative values")
    # Volume is radius cubed * pi * 4/3
    return pow(radius, 3) * pi * 4 / 3
```

File: maths/volume.py [hunk 7]
```diff
@@ -284,6 +288,7 @@ def vol_hemisphere(radius: float) -> float:
         ...
     ValueError: vol_hemisphere() only accepts non-negative values
     """
+    # Volume - 4/3 * pi * radius cubed
     if radius < 0:
         raise ValueError("vol_hemisphere() only accepts non-negative values")
     return 2 / 3 * pi * pow(radius, 3)
```
Comment (cclauss) [inline maths/volume.py:294]:
```suggestion
    if radius < 0:
        raise ValueError("vol_hemisphere() only accepts non-negative values")
    # Volume is radius cubed * pi * 2/3
    return pow(radius, 3) * pi * 2 / 3
```

File: maths/volume.py [hunk 8]
```diff
@@ -310,6 +315,7 @@ def vol_circular_cylinder(radius: float, height: float) -> float:
         ...
     ValueError: vol_circular_cylinder() only accepts non-negative values
     """
+    # Volume - radius squared * height * pi
     if height < 0 or radius < 0:
         raise ValueError("vol_circular_cylinder() only accepts non-negative values")
     return pi * pow(radius, 2) * height
```
Comment (cclauss) [inline maths/volume.py:321]:
```suggestion
    if height < 0 or radius < 0:
        raise ValueError("vol_circular_cylinder() only accepts non-negative values")
    # Volume is radius squared * height * pi
    return pow(radius, 2) * height * pi
```

File: maths/volume.py [hunk 9]
```diff
@@ -378,6 +385,7 @@ def vol_conical_frustum(height: float, radius_1: float, radius_2: float) -> floa
         ...
     ValueError: vol_conical_frustum() only accepts non-negative values
     """
+    # Volume - 1/3 * pi * height * (radius_1 squared + radius_2 squared + radius_1 * radius_2)
```
Comment (cclauss) [inline maths/volume.py:388]: Tests are failing on:
* maths/volume.py:388:89: E501 __line too long__ (94 > 88 characters)

Also, please move this line down to be just above the calculation.  Make sure that the order of the values in the comment and in the calculation are the same.

**Review by cclauss — CHANGES_REQUESTED** [review]
I have moved the comments down to be just above the calculations.  I have changed the order of the terms to be the same in both the comments and the calculations.  I have changed `Volume -` to `Volume is` because `-` has a special meaning in calculations.  ;-)

Please fix the line that is failing our tests because it has more than 88 characters.

File: maths/volume.py [hunk 10]
```diff
@@ -344,6 +348,7 @@ def vol_hollow_circular_cylinder(
         ...
     ValueError: outer_radius must be greater than inner_radius
     """
+    # Volume - (outer_radius squared - inner_radius squared) * pi * height
```
Comment (cclauss) [inline maths/volume.py:351]: Please move this comment down to be just above the calculation to make it easier for the reader to understand.

File: maths/volume.py [hunk 11]
```diff
@@ -263,7 +264,8 @@ def vol_sphere(radius: float) -> float:
     """
     if radius < 0:
         raise ValueError("vol_sphere() only accepts non-negative values")
-    return 4 / 3 * pi * pow(radius, 3)
+    # Volume is radius cubed * pi * 4/3
+    return pow(radius, 3) * pi * 4 / 3
```
Comment (cclauss) [inline maths/volume.py:268]:
```suggestion
    # Volume is 4/3 * pi * radius cubed
    return  4 / 3 * pi * pow(radius, 3)
```

**Review by Aadi077 — CHANGES_REQUESTED** [review]
Change "volume -" to "volume is" because - has another meaning which would not apply.

**Review by advik-student-dev — COMMENTED** [review]
thank u for the changes!

**Comment by advik-student-dev** [comment]
how do i commit your changes?

**Comment by raghavasrujan** [comment]
Hello Sir, Can you assign this to me?

**Review by cclauss — APPROVED** [review]

**Pull request merged**